pythonProject1/main.py

```diff
@@ -1,36 +1,49 @@
 import math
-#def is_prime(number: int):
+TARGET_NUMBER = 13195
 
+def largest_prime_factor(subject: int)-> int:
+    minimal_possible_answer = round(subject / 2)
+    multiplier = minimal_possible_answer
+    for y in reversed(range(minimal_possible_answer)):
 
-subject = 600851475143
-solution = 9
-answer = 0
-is_primal = True
-primal_checker = 0
-while solution < subject:
-    primal_checker = solution - 1
+        for y in reversed(range(minimal_possible_answer)):
+            if minimal_possible_answer * multiplier == subject and is_prime(minimal_possible_answer):
```

Comment:
You should use for loops in a different manner. 
use the iterator in the function(y) - do no use what you iterating on (minimal_possible_answer)